include/libtorrent/session_settings.hpp

```diff
@@ -152,6 +152,12 @@ namespace libtorrent
 		// 5.
 		int urlseed_pipeline_size;
 
+		// the maximum request range of an url seed in bytes. This value defines
+		// the largest possible sequentiel web seed request. Default is 16777216
+		// (16 * 1024 * 1024). Lower values are possible but will be ignored if
+		// if they are lower then piece size.
+		int urlseed_max_request_bytes;
```

Comment:
this structure, ``session_settings`` is deprecated. please don't add new fields to it. instead, in the settings_pack definition, use ``SET_NOPREV`` (is in there's no previous field)